PowerEditor/installer/nativeLang/romanian.xml

```diff
@@ -1059,16 +1071,17 @@
                     <Item id="6293" name="Lățime constantă"/>
                     <Item id="6207" name="Semn de carte pentru afișaj"/>
                     <Item id="6223" name="Istoricul schimbărilor afișajului"/>
+                    <Item id="6296" name="Afișează în text"/>
                     <Item id="6211" name="Setările vârfurilor verticale"/>
                     <Item id="6213" name="Modul fundal"/>
-                    <Item id="6237" name="Se adaugă marcatorul de coloană indicând poziția acestuia cu un număr zecimal.
-Se pot defini mai mulți marcatori de coloană utilizând spațiul alb pentru a separa diferitele numere."/>
                     <Item id="6231" name="Lățimea chenarului"/>
                     <Item id="6235" name="Fără vârf"/>
                     <Item id="6208" name="Umplutură"/>
                     <Item id="6209" name="Stânga"/>
                     <Item id="6210" name="Dreapta"/>
                     <Item id="6212" name="Fără distracție"/>
+		    <Item id="6295" name="Istoricul schimbărilor"/>
+		    <Item id="6296" name="Afișează în text"/>
```

Comment:
id 6296 is repeated.

Reply:
I solved it. I removed the repetead entry.